### PR TITLE
Wrapper for rest calls

### PR DESCRIPTION
**Pull request by yurypat**

### CODE REVIEW

File: src/Mews.Fiscalization.Greece/AadeLogger.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            OnError = onError;
+            OnInfo = onInfo;
+            OnDebug = onDebug;
+        }
+
+        public AadeLogger(Action<string, object> eagerLogger)
+        {
+            OnError = eagerLogger;
+            OnInfo = eagerLogger;
+            OnDebug = eagerLogger;
```
Comment (KaliCZ) [inline src/Mews.Fiscalization.Greece/AadeLogger.cs:18]: Can we instead forward this constructor to the other constructor?

File: src/Mews.Fiscalization.Greece.Tests/UnitTests/OutboundRequestSerializationTests.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                .Build<Tax>()
+                .With(x => x.TaxCategory, 1)
+                .With(x => x.LineNumber, 3)
+                .Create();
+
+            var invoice = _fixture.Build<Invoice>()
+                .With(x => x.Taxes, new[] { taxes })
+                .With(x => x.InvoiceDetail, invoiceDetail)
+                .With(x => x.InvoiceSummary, invoiceSummary)
+                .Create();
```
Comment (KaliCZ) [inline src/Mews.Fiscalization.Greece.Tests/UnitTests/OutboundRequestSerializationTests.cs:79]: In the future, let's replace this with some kind of real usage of models. (I know you probably intend to do it, just commenting.)

**Review by KaliCZ — APPROVED** [review]

File: src/Mews.Fiscalization.Greece.Tests/UnitTests/OutboundRequestSerializationTests.cs (hunk 2 above)
Comment (yurypat) [reply src/Mews.Fiscalization.Greece.Tests/UnitTests/OutboundRequestSerializationTests.cs:79]: Yes, we will do it later. Thanks for your comment

File: src/Mews.Fiscalization.Greece/AadeLogger.cs (hunk 1 above)
Comment (yurypat) [reply src/Mews.Fiscalization.Greece/AadeLogger.cs:18]: Yes, thanks for noticing this. I will change it in the next PR

**Pull request merged**